tests/function-controller/pkg/shared/shared.go

```diff
@@ -1,5 +1,25 @@
 package shared
 
+import (
+	"errors"
+	"time"
+
+	"k8s.io/client-go/dynamic"
+)
+
 type Logger interface {
 	Logf(format string, args ...interface{})
 }
+
+var (
+	ErrInvalidDataType = errors.New("invalid data type")
+)
+
+type Container struct {
+	DynamicCli  dynamic.Interface
+	Namespace   string
+	WaitTimeout time.Duration
+	Kind        string
```

Comment:
Kind? I don't see any usage of this field.

Reply:
I'll take care of this in next PR